addons/bus/models/ir_async.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import logging
+from odoo import models, fields
+
+_logger = logging.getLogger(__name__)
+
+
+class IrAsync(models.Model):
+    _inherit = "ir.async"
+
+    notify = fields.Boolean(default=False)
+
+    def call_notify(self, target, args=None, kwargs=None, description=False):
+        """
+        Thread/Process-like API to create asynchronous jobs.
+
+        The method will be called later as ``target(*args, **kwargs)`` in a
+        new transaction using a copy of the current environment (user, context,
+        recordset).  Arguments must be serializable in JSON.
+
+        The notifications are:
+
+        - created, the task has been enqueued
+        - processing, the worker begins to process the task
+        - succeeded, the task succeeded with a result the user must process
+        - failed, when the task failed with an error
+        - done, the task succeeded without result or the user processed it
+
+        See the ``async_job`` javascript service.
+
+        The ``description`` is used to list the task in the systray menu
+        in the frontend. You may omit it to hide the task from the
+        systray but still get the notifications.
+        """
+        job = self.call(target, args, kwargs)
+        job.name = description
+        job.notify = True
```

Comment:
`self.call` will perform a `self.create`, so why not passing `name` and `notify` as default values?
```suggestion
        job = self.with_context(name=description, notify=True).call(target, args, kwargs)
```

Reply:
And I suppose I should remove those two context key from the context attached to the job ?

Reply:
Oops, I actually meant
```suggestion
        job = self.with_context(default_name=description, default_notify=True).call(target, args, kwargs)
```